netdisco/discoverables/webos_tv.py

```diff
@@ -8,8 +8,12 @@ class Discoverable(SSDPDiscoverable):
     """Add support for discovering LG WebOS TV devices."""
 
     def info_from_entry(self, entry):
-        """Return the most important info from a uPnP entry."""
-        return urlparse(entry.values['location']).hostname
+        """Get most important info, which is name, model and host."""
+        name = entry.description['device']['friendlyName']
+        model = entry.description['device']['modelName']
+        host = urlparse(entry.values['location']).hostname
+
+        return name, model, host
```

Comment:
Can u give me an example thats it already doing that ?
Ps: im not a pythong developer and by disct I guess u're referring to an key->value associative array, same way I did for daikin ?